### ISSUE TITLE
on import in appModule: forRoot is not a function

### ISSUE THREAD

**Issue by wcjord** [valor-software/ng2-dragula#937]
I want to import ng2-dragula in my appModule as described in the readme.

```
import { DragulaModule } from 'ng2-dragula';

@NgModule({
  declarations: [
    ...
  ],
  imports: [
   ...
    DragulaModule.forRoot(),
],
...
```

However, I get the error:

`__WEBPACK_IMPORTED_MODULE_18_ng2_dragula__["DragulaModule"].forRoot is not a function
`
I can avoid the error by removing ".forRoot()" but things aren't working as advertised farther down the line so I'd like to solve this issue to make sure it's not causing the problems.

Mine is an Ionic Cordova project with the following info:
`Ionic:

   ionic (Ionic CLI)  : 4.2.1
   Ionic Framework    : ionic-angular 3.9.2
   @ionic/app-scripts : 3.2.0

Cordova:

   cordova (Cordova CLI) : 8.1.2 (cordova-lib@8.1.1)
   Cordova Platforms     : android 7.1.4, browser 5.0.3, ios 4.5.5
   Cordova Plugins       : no whitelisted plugins (17 plugins total)

System:

   Android SDK Tools : 26.1.1 (/Users/willjc/Library/Android/android-sdk)
   ios-deploy        : 2.0.0
   NodeJS            : v8.11.3 (/usr/local/bin/node)
   npm               : 5.6.0
   OS                : macOS
   Xcode             : Xcode 10.1 Build version 10B61`

Has anyone seen this error before?

**Comment by cormacrelf** [comment]
Sorry, not relevant. Problem's on your end.

**Comment by wcjord** [comment]
I don't know whose fault it was. I ended up just using this to import (so minus the .forRoot()).

```
import { DragulaModule } from 'ng2-dragula';

@NgModule({
  declarations: [
    ...
  ],
  imports: [
   ...
    // DragulaModule.forRoot(),
   DragulaModule
],
...
```

And we got the downstream issues figured out which were related to using css within a component. We put dragula css inside a component that we thought would encapsulate the whole operation but it didn't get applied to the placeholder and shadow elements that are created when you drag. 

So yeah, importing without .forRoot() works fine.
